src/test/resources/data/InstructorCourseStudentDetailsPageUiTest.json

```diff
@@ -91,24 +114,60 @@
         "sectionLevel": {},
         "sessionLevel": {}
       }
+    },
+    "instructor1OfTestingSanitizationCourse": {
+      "googleId": "idOfTestingSanitizationCourse",
+      "courseId": "idOfTestingSanitizationCourse",
+      "name": "Instructor<script> alert('hi!'); </script>",
+      "email": "instructor1@sanitization.tmt",
+      "isArchived": false,
+      "role": "Co-owner",
+      "isDisplayedToStudents": true,
+      "displayedName": "inst<script>alert('hi!');</script>",
+      "privileges": {
+        "courseLevel": {
+          "canviewstudentinsection": true,
+          "cangivecommentinsection": true,
+          "cansubmitsessioninsection": true,
+          "canmodifysessioncommentinsection": true,
+          "canmodifycommentinsection": true,
+          "canmodifycourse": true,
+          "canviewsessioninsection": true,
+          "canmodifysession": true,
+          "canviewcommentinsection": true,
+          "canmodifystudent": true,
+          "canmodifyinstructor": true
+        },
+        "sectionLevel": {},
+        "sessionLevel": {}
+      }
     }
   },
   "students": {
     "registeredStudent": {
       "googleId": "CCSDetailsUiT.jose.tmms",
       "email": "CCSDetailsUiT.jose.tmms@gmail.tmt",
       "course": "CCSDetailsUiT.CS2104",
-      "name": "José Gómez</option></td></div>'\"",
```

Comment:
These are valid case `</option></td></div>'\"` and `</td></div>'\"`. You should include them in `student1InTestingSanitizationCourse`.
I would say sanitization is not only to prevent script injection, but also to make sure everything display correctly (e.g. the `<div>` tag closes properly).